tools/cldr-code/src/main/resources/org/unicode/cldr/util/data/Locales.txt

```diff
@@ -886,6 +886,8 @@ Mozilla	;	und	;	modern
 
 Rodakych ; pcm ; modern ; Nigerian Pidgin
 
+rohingyazuban	;	rhg	;	modern
```

Comment:
LGTM but do we need to have the script? are they covering rhg_Arab and rhg_Rohg ?

Reply:
I figured by leaving it unstated they would be able to work with both?

Reply:
The default script is Rohg, so unspecified implies that I think.